README.md

```diff
@@ -15,3 +15,9 @@ Trading events per event:
 3. Buy or Sell delta to 0 via TWAP if necessary
 4. Place bid & ask at gamma levels
 5. Start Scalper Window timer
+
+Environment Variables
+DEV=true || DEV=false
+RPC=[Enter RPC url]
+SOL, BTC, ETH, MNGO = [OFF || ON,DELTA_OFFSET,THEO_VOL]
+Example: DEV=false RPC=https://api.mainnet-beta.solana.com SOL=ON,-117.7,0.84 BTC=OFF ETH=OFF MNGO=ON,0,1.6 yarn run main
```

Comment:
now that vol is hidden, can we remove the randomness added in router when determining if a bid exceeds threshold?
does it make sense to have both uses of theo vol still be the same constant, or should they be different?

Reply:
We still want it or at least I don't see an issue keeping it. The implied vol can still be backed out. 

I am thinking that this threshold price we may want to use the Backstop IV not Theo IV however. As it's unlikely that an MM will bid above what the theo IV in the market. Probably rename this parameter to HEDGE_VOL

Reply:
https://github.com/Dual-Finance/risk-manager/issues/78

Reply:
Not an issue, but if the randomness does become unnecessary, should be removed for the sake of code simplicity and maintainability. Good for now though.